### ISSUE TITLE
Convert to Frequency modification silently drops trip patterns

### ISSUE THREAD

**Issue by Nate-Wessel** [conveyal/analysis-ui#1009]
With the default settings, the _convert to frequency_ modification seems to silently drop trip patterns that are not explicitly replaced by a timetable. No error, no warning, no red lines on the map, etc. It's hard to tell they are gone except that the accessibility is just not there in analysis mode. 

This could cause mistakes if you have one or two trip patterns. With more, it would be positively difficult to get the settings right and double check that all the trip patterns are properly covered. My regular route in Cincinnati way back when has four branches and seems to use 17 trip patterns now. 

If this is a reasonable default behaviour, I would suggest showing all un-timetabled trip patterns on the map in red just like for the _remove trip pattern_ modification. Beyond that, it might be good to display a list of these to-be-removed trip patterns by name, so the user can use that as a sort of checklist for the timetables they still need to make. 

Related to #1008 & #1004

**Comment by Nate-Wessel** [comment]
I'm also finding that a timetable does not seem able to apply to more than one trip pattern, though it's possible to select any number of them. You still only get to select one template trip, but how this would apply to multiple trip patterns is not clear and apparently undefined. 

Again, whichever trip pattern the timetable happens not to apply to seems to get dropped without errors or warnings. This is true even if the "retain existing trips option is selected". 

To test this, find a route with two opposite trip patterns (e.g. CTRAN's 7) and apply a timetable to both patterns. Only one direction will have service.

**Comment by ansoncfit** [comment]
@Nate-Wessel could you confirm that we could resolve this issue with these two changes?
1. Rename "Convert to Frequency" modification to "Replace Timetables" in the UI and user manual
2. Change the pattern selector in each frequency entry to allow selecting only a single pattern

**Comment by Nate-Wessel** [comment]
@ansoncfit those would be definite improvements, but I'm not sure it would totally resolve this. 
I think the main problem is that all other trip patterns on a route get removed (assigned no timetable) if any one trip pattern on that route is assigned a replacement timetable. IMO, the modification shouldn't effect timetables/patterns that aren't explicitly modified. If one wanted to remove those trip patterns they should use the Remove Trip Patterns mod, and then they'll show up on the map in red.

Alternatively, if we keep the current behaviour, we should explicitly list and/or show trip patterns that are removed by the modification (i.e. given no timetable) and update that when a new replacement timetable is added/removed.

Basically, the whole effect of the modification on the route should be clearly visible.

**Comment by mattwigway** [comment]
I'm the one who wrote this modification initially, I believe, and the idea behind this was that you would want to translate "route X runs every 15 minutes" into a modification, and you wouldn't want to leave behind timetabled school trips that happen once a day, etc.

**Comment by abyrd** [comment]
We may not want to actually change the behavior of the modifications, but the behavior of this modification type is non-obvious and should be verified and better documented.
